src/App.js

```diff
@@ -1,25 +1,35 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import createDigits from './components/calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="calculator">
+      <div className="display">
+        <span>0</span>
+      </div>
+
+      <div className="transform">
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
+        <button type="button">%</button>
+      </div>
+
+      <div className="digits">
+        { createDigits() }
+      </div>
+
+      <div className="zero_digits">
+        <button type="button" className="zero">0</button>
+        <button type="button" className="point">.</button>
+      </div>
+
+      <div className="operators">
+        <button type="button">&#xF7;</button>
+        <button type="button">*</button>
+        <button type="button">-</button>
+        <button type="button">+</button>
+        <button type="button">=</button>
```

Comment:
- [ ] [OPTIONAL] If you'd like, please create a reusable `Button` component since you have a lot of buttons. This will make your work a lot better in my opinion 👍